core/arch/arm/plat-qcom/main.c

```diff
@@ -34,6 +35,17 @@ void plat_console_init(void)
 	register_serial_console(&console_data.chip);
 }
 
+static TEE_Result platform_banner(void)
+{
+	uint32_t major = 0, minor = 0, step = 0;
+
+	qti_ramblur_pimem_get_version(&major, &minor, &step);
+	IMSG("RAMBLUR %d.%d.%d", major, minor, step);
+
+	return TEE_SUCCESS;
+}
+service_init(platform_banner);
```

Comment:
Rather than a separate `service_init()`, this should go to driver init as well. I would prefer it to be a `DMESG` since it might not be useful to the user apart from driver debug itself.